portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,28 +1,43 @@
 package com.google.sps.data;
 
-import java.time.LocalDateTime;
+import com.google.appengine.api.datastore.Entity;
 
 /** Represents a comment with related details. */
-public final class Comment {
+public final class Comment implements EntityConvertible {
     private final String commenter;
     private final String content;
-    private final LocalDateTime time;
+    private final long timestamp;
 
-    public Comment(String commenter, String content, LocalDateTime time) {
+    public Comment(String commenter, String content, long timestamp) {
         this.commenter = commenter;
         this.content = content;
-        this.time = time;
+        this.timestamp = timestamp;
     }
 
-    public String getCommenter() {
-        return commenter;
+    static class Constants {
```

Comment:
Maybe rename to PropertyName?
static final already means constant.

Reply:
Fixed